### DISCUSSION TITLE
Libvips php. I need cut a corner and make it transparent

### DISCUSSION THREAD

**Discussion by cmsaccount** [libvips/libvips#3813] [Help wanted]
Hello,

Could you help me i need cut a corner from an image and make it transparent? in Libvips php
![2](https://github.com/libvips/libvips/assets/51406194/156c9eeb-7bce-4e6b-845b-df0d293f5be6)

**Comment by jcupitt** [comment]
Hi @cmsaccount,

I would probably just clip with an SVG.

There's a pyvips example here: https://github.com/libvips/libvips/discussions/2271
